allensdk/brain_observatory/behavior/behavior_session.py

```diff
@@ -1360,6 +1361,7 @@ def _read_stimuli(
         sync_file: Optional[SyncFile],
         monitor_delay: float,
         stimulus_presentation_columns: Optional[List[str]] = None,
+        trials: Optional[Trials] = None,
```

Comment:
why is it defined as optional, since it looks like it is always provided.

Reply:
Mostly I wanted to keep the inputs the same order and keep test behavior roughly the same.

Reply:
If it is required then we need to make the necessary changes. If it is defined as optional then it looks like it is not needed, when I think your intention is to always add the `trials_id`.

Reply:
Thanks, missed this one.